src/dmd/expressionsem.d

```diff
@@ -5103,7 +5103,14 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             }
 
             checkFunctionAttributes(exp, sc, exp.f);
-            checkAccess(exp.loc, sc, null, exp.f);
```

Comment:
this check is useless. It just returns `false` all the time, it never issues any errors.